app-crypt/qca/qca-9999.ebuild

```diff
@@ -31,80 +26,56 @@ RDEPEND="
 		libressl? ( dev-libs/libressl )
 		dev-libs/pkcs11-helper
 	)
-	qt4? ( dev-qt/qtcore:4 )
-	qt5? (
-		dev-qt/qtcore:5
-		dev-qt/qtconcurrent:5
-		dev-qt/qtnetwork:5
-	)
 	sasl? ( dev-libs/cyrus-sasl:2 )
 	ssl? (
 		!libressl? ( >=dev-libs/openssl-1.0.1:0= )
 		libressl? ( dev-libs/libressl:= )
 	)
 "
-DEPEND="${RDEPEND}
+DEPEND="${COMMON_DEPEND}
+	dev-qt/qtnetwork:5
```

Comment:
build-time dep only?

Reply:
actually, it seems only to be required with `USE=test`, going to fix upstream first. Otherwise it built fine without finding Network